src/format/command.rs

```diff
@@ -6,21 +6,29 @@ use std::str::FromStr;
 
 #[derive(Debug, PartialEq)]
 pub struct Formatted {
-    pub path: PathBuf,
+    pub source: PathBuf,
+    pub destination: FormatDestination,
     pub input: String,
     pub output: String,
 }
 
+#[derive(Debug, PartialEq)]
+pub enum FormatDestination {
```

Comment:
Which is a more idiomatic name in Rust: `FormatDestination` or just `Destination`?

Reply:
No idea :) Do what you prefer!